packages/storybook8/stories/Components/TypingIndicator/index.stories.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { TypingIndicator as TypingIndicatorComponent } from '@azure/communication-react';
+import { Meta } from '@storybook/react';
+import { defaultTypingUsers, hiddenControl } from '../../controlsUtils';
+import { CustomStylingExample } from './snippets/CustomizeStyle.snippet';
+import { CustomUserRenderExample } from './snippets/CustomizeUserRendering.snippet';
+import { TypingIndicatorExample } from './snippets/TypingIndicator.snippet';
+export { TypingIndicator } from './TypingIndicator.story';
+
+export const TypingIndicatorExampleDocsOnly = {
+  render: TypingIndicatorExample
+};
+
+export const CustomizeStyleExampleDocsOnly = {
+  render: CustomStylingExample
+};
+
+export const CustomizeUserRenderingExampleDocsOnly = {
+  render: CustomUserRenderExample
+};
+
+const meta: Meta = {
+  title: 'Components/Typing Indicator',
+  component: TypingIndicatorComponent,
+  argTypes: {
+    // Hidden Controls
```

Comment:
add typing users from "controls to add" to fix the naming from `typingUsers` to `Typing Users`

Reply:
Added